Dockerfile

```diff
@@ -1,24 +1,43 @@
 FROM node:lts-alpine
-ENV NPM_CONFIG_LOGLEVEL warn
 
-# Install "git"
-RUN apk update \
- && apk add alpine-sdk
+ARG VERSION_INFO=noVersion
+
+ENV NPM_CONFIG_LOGLEVEL warn
+ENV PORT=1337
 
 # Set working dir as /app
 WORKDIR /app
 
-# Add sources to /app
-COPY . .
-RUN adduser -S dr4ftuser
-RUN chown dr4ftuser -R .
```

Comment:
Running `chown` manually like this has a sneaky behavior where it actually copies files in layers above it, increasing the size of the image. The `COPY` instruction includes a `--chown` flag that does not trigger this behavior.